src/applications/pact-act/utilities/page-setup.js

```diff
@@ -32,11 +32,11 @@ export const applyFocus = (parentId, headerHasFocused, setHeaderHasFocused) => {
         header?.focus();
         setHeaderHasFocused(true);
       }
-    }, 500);
+    }, 150);
```

Comment:
I was seeing a memory leak error when navigating quickly between questions. Because the TernaryRadios component runs an `applyFocus` function when the radios are loaded and had this half-second day, sometimes I was clicking nav buttons faster than the half-second, and the memory leak happened. Shortening this still keeps the focus behavior and tidies up the memory leak.